src/OSPSuite.InstallationValidator.Core/Domain/StartableProcess.cs

```diff
@@ -46,14 +46,15 @@ public void Dispose()
 
       public virtual void Wait(CancellationToken token)
```

Comment:
Well I ma looking at this SafeWaitHandle and it says that it is not recommended to set the second parameter ownHandle to false. Why did you use this value?